nms/app/views/equipment/FEGGatewayTable.tsx

```diff
@@ -209,3 +244,103 @@ function GatewayStatusTable(props: WithAlert & {refresh: boolean}) {
   );
 }
 const StatusTable = withAlert(GatewayStatusTable);
+
+type EquipmentGatewayUpgradeType = {
+  name: string;
+  id: GatewayId;
+  hardwareId: string;
+  tier: string;
+  currentVersion: string;
+};
+function UpgradeTable() {
```

Comment:
This is in largely a copy of the `EquimentGateway.UpgradeTable`. That is a bit sad, but I understand that it might be    somewhat awkward to abstract this. Have you tried?